Bucketeer2/Sources/Internal/Remote/ApiClientImpl.swift

```diff
@@ -0,0 +1,128 @@
+import Foundation
+
+final class ApiClientImpl: ApiClient {
+
+    static let DEFAULT_REQUEST_TIMEOUT_MILLIS: Int64 = 30_000
```

Comment:
This is the same value as https://github.com/bucketeer-io/android-client-sdk/blob/37b0a1a03461693e6b7b608de058bce5c2a8a838/bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/remote/ApiClientImpl.kt#L24.